packages/fxa-auth-server/config/index.ts

```diff
@@ -11,6 +11,41 @@ const DEFAULT_SUPPORTED_LANGUAGES = require('./supportedLanguages');
 convict.addFormats(require('convict-format-with-moment'));
 convict.addFormats(require('convict-format-with-validator'));
 
+function makeMySQLConfig(envPrefix: string, database: string) {
```

Comment:
I thought this function was exported somewhere in `fxa-shared`?

Reply:
No, `makeMySQLConfig` wasn't exported to `fxa-shared` just the interface `MySQLConfig` was extracted. There was some issue with exporting/importing `makeMySQLConfig`, that would result in an error when trying to use it to generate the configs: `makeMySQLConfig is not a function`

Reply:
Was `makeMySQLConfig` undefined or something else, since it wasn't a function?

Reply:
It was defined and exported, but I was getting a typescript compilation error

Reply:
I'll give it another go and see if I get the same issue